src/openforms/prefill/tests/test_prefill_hook.py

```diff
@@ -219,6 +220,32 @@ def get_prefill_values(submission, attributes):
         self.assertIsNotNone(field["defaultValue"])
         self.assertEqual(field["defaultValue"], "some-default")
 
+    def test_logging_for_empty_prefill(self):
+        config = deepcopy(CONFIGURATION)
+        form_step = FormStepFactory.create(form_definition__configuration=config)
+        submission = SubmissionFactory.create(form=form_step.form)
+
+        register = Registry()
+
+        @register("demo")
+        class EmptyPrefillPlug(DemoPrefill):
+            @staticmethod
+            def get_prefill_values(submission, attributes):
+                return {}
+
+        apply_prefill(
+            configuration=config,
+            submission=submission,
+            register=register,
+        )
+
+        logs = TimelineLogProxy.objects.all()
+
+        self.assertNotEqual(logs.count(), 0)
```

Comment:
either use `self.assertTrue(logs.exists())` or an explicit count: `self.assertEqual(logs.count(), 1)`

is there a reason to expect more than one log entry here?